### PR TITLE
[12.0][MIG] delivery_carrier_label_default: migration

### PR DESCRIPTION
**Pull request by i-vyshnevska**

### CODE REVIEW

**Review by Tonow-c2c — APPROVED** [review]
Code review only

File: delivery_carrier_label_default/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* Yannick Vaucher <yannick.vaucher@camptocamp.com>
+* Sébastien Alix <sebastien.alix@camptocamp.com>
+* Sergio Teruel <sergio.teruel@tecnativa.com>
```
Comment (yvaucher) [inline delivery_carrier_label_default/readme/CONTRIBUTORS.rst:3]: @i-vyshnevska please add yourself

**Review by yvaucher — APPROVED** [review]
LGTM

File: delivery_carrier_label_default/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (i-vyshnevska) [reply delivery_carrier_label_default/readme/CONTRIBUTORS.rst:3]: updated

**Comment by rousseldenis** [comment]
@i-vyshnevska Could you see travis ?

**Comment by i-vyshnevska** [comment]
@rousseldenis updated and green
@yvaucher @florian-dacosta can you pass review on this, I did some refactoring to https://github.com/OCA/delivery-carrier/pull/220

File: base_delivery_carrier_label/models/stock_picking.py [hunk 2]
```diff
@@ -106,6 +107,7 @@ def action_generate_carrier_label(self):
             shipping_labels = pick.generate_shipping_labels()
             for label in shipping_labels:
                 data = pick.get_shipping_label_values(label)
+                data['datas'] = base64.b64encode(data['datas'])
```
Comment (florian-dacosta) [inline base_delivery_carrier_label/models/stock_picking.py]: I don't get why you encode in base64 back when we just removed it some days ago?!
Usually carrier already return the label in base64, so base_delivery_carrier_label now expect the label data to be in base64 format already. 
You shoud encode the label directly in the module delivery_carrier_label_default.
In the method `generate_default_label` on `stock.picking`

Else it seems good to me

**Review by florian-dacosta — CHANGES_REQUESTED** [review]

File: base_delivery_carrier_label/models/stock_picking.py (hunk 2 above)
Comment (i-vyshnevska) [reply base_delivery_carrier_label/models/stock_picking.py]: yep, you are correct I missed this, updated

**Comment by rousseldenis** [comment]
@florian-dacosta Ok with this version ?

**Review by florian-dacosta — APPROVED** [review]

**Comment by florian-dacosta** [comment]
yes, thanks @i-vyshnevska

**Pull request merged**